.github/workflows/test.yml

```diff
@@ -50,15 +89,16 @@ jobs:
           mkdir qemu-data
```

Comment:
```suggestion
          mkdir -p qemu-data
```

Currently this line would fail if folder already exists.

Reply:
0fb47d0128d6b1601e1d9bc018b68cf76406519f